packages/icons/src/hdmi.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
@Tigge added a license comments to all(?) icons in #37. Should that be done here as well?

Reply:
Yes!